changelog/fragments/k8s-1.24-bump.yaml

```diff
@@ -0,0 +1,130 @@
+    migration:
+      header: Go (go/v3) and Hybrid Helm (hybrid.helm/v1-alpha) language based operators - Add support for Go 1.18
+      body: |
+        To update your project to use Go 1.18:
+
+        1. In the `go.mod` file:
+
+        - Replace `go 1.17` with `go 1.18`   
+
+        2. Run `go mod tidy` to ensure all dependencies are upgraded
+
+  - description: >
+      For `operator-sdk bundle validate` command and GoodPractices validator (`--select-optional name=good-practices`): fix channel naming validation [More info](https://github.com/operator-framework/api/pull/231)
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "bugfix"
+
+    # Is this a breaking change?
+    breaking: false
+
+  - description: >
+      For `operator-sdk bundle validate` command and GoodPractices validator (`--select-optional name=good-practices`): Add CRD description check validation [More info](https://github.com/operator-framework/api/pull/234)
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+  - description: >
+      For `operator-sdk bundle validate` command and GoodPractices validator (`--select-optional name=good-practices`): Add a new check to warn authors when permissions to create CRDs are found [More info](https://github.com/operator-framework/api/pull/241)
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+  - description: >
+      For Helm (helm/v1) and Hybrid Helm (hybrid.helm/v1-alpha) language based operators: Upgrade helm from `v3.6.2` to `v3.9.0`
+    breaking: false
+
+  - description: >
+      For Ansible (ansible/v1) and Helm (helm/v1) language based operators: change the default value for the manager.option LeaderElectionResourceLock from `comfigmaps` to `configmapsleases` to allow a safe transition to use the leases option on the future. [More info](https://github.com/kubernetes-sigs/controller-runtime/pull/1903)
+    breaking: false
+  
+  - description: >
+      For Quarkus (quarkus/v1-alpha) language based operator: add bundle generation [More info](https://github.com/operator-framework/java-operator-plugins/pull/84)
+    breaking: false
+  
+  - description: >
+      For Quarkus (quarkus/v1-alpha) language based operator: upgrade dependency from `v0.0.3` to `v0.5.1` [More info](https://github.com/operator-framework/java-operator-plugins/releases)
+    breaking: false
+  
+  - description: >
+      For Helm Hybrid (hybrid.helm/v1-alpha) language based operator: Upgrade dependency from `v0.0.10` to `v0.0.11`. [More info](https://github.com/operator-framework/helm-operator-plugins/releases)
+    breaking: false  
```

Comment:
add space at the end

Reply:
nit

Reply:
commit https://github.com/operator-framework/operator-sdk/pull/5843/commits/b30c7a866b98ba3603f840af06110339c21aeb91